src/app/components/merge/MergeModalContent.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Alert } from 'react-components';
+
+import MergeTable from './MergeTable';
+
+const MergeModalContent = ({
+    onSortEnd,
+    orderedContacts,
+    isChecked,
+    isDeleted,
+    onClickCheckbox,
+    onClickDetails,
+    onClickDelete,
+    onClickUndelete,
+    onClickPreview
+}) => {
+    return (
+        <>
+            <Alert>
+                {c('Description')
+                    .jt`Use Drag and Drop to rank merging priority between contacts. Uncheck the contacts you do ${(
+                    <b key="boldface">not</b>
+                )} want to merge`}
+            </Alert>
+            <Alert type="warning">
+                {c('Description')
+                    .t`You can mark for deletion the contacts that you do not want neither to merge nor to keep.
+                    Deletion will only take place after the merge button is clicked.`}
+            </Alert>
+            <MergeTable
```

Comment:
You can use the slots pattern and pass `<MergeTable` to this component to avoid have to drill props like this

Reply:
So you'd just render `{mergeTable}` here